locales/en.json

```diff
@@ -65,7 +65,8 @@
       }
     },
     "outcomes": {
-      "deleted": "$t({{ name }}) deleted successfully."
+      "deleted": "$t({{ name }}) deleted successfully.",
+      "disabled": "$t(Default ordering {{ name }}) disabled successfully."
```

Comment:
easy fix - $t() translates whatever it wraps, so it should only wrap `{{ name }}`